src/App.jsx

```diff
@@ -5,18 +5,19 @@ import Header from './components/Header'
 import Nav from './components/Nav'
 import Home from './components/Home'
 import ArticleList from './components/ArticleList'
+import Article from './components/Article'
 
 function App() {
-
+  const [articles, setArticles] = useState([])
```

Comment:
Do we need this state to be set this high? We are only passing it down to one component currently, which is an indicator that it could probably be set lower. 'As low as possible, but as high as it needs to be'.